packages/common-all/src/parse.ts

```diff
@@ -26,7 +23,7 @@ export const parse = <T extends z.ZodTypeAny>(
       new DendronError({
         message: [
           ...(msg ? [msg] : []),
-          JSON.stringify(formatErrors(parsed.error.format()), null, 2),
```

Comment:
Simplifying here the formatting. Lets return to formatting at a later moment where we actually define how that should look like.